packages/squiggle-lang/package.json

```diff
@@ -11,6 +11,7 @@
     "test": "jest",
     "test:watch": "jest --watchAll",
     "coverage": "rm -f *.coverage; yarn clean; BISECT_ENABLE=yes yarn build; yarn test; bisect-ppx-report html",
+    "reducer:format": "find src/rescript/Reducer src/rescript/ReducerInterface -type f \\( -name '*.res' -o -name '*.resi' \\) -exec sh -c 'bsc -format {} | sponge {}'  \\;",
```

Comment:
we're gonna have people `rescript format -all` moving forward, and perhaps a script in `package.json` for it. but thanks!

Reply:
It's still not clear to me how you're getting `bsc -format` to work, because [the `-format` flag was taken out of `bsc` in `9.1`](https://forum.rescript-lang.org/t/rescript-9-1-how-can-we-format-to-standard-out/1590). `rescript format -stdin .res $file` is equivalent to what that functionality was.

Reply:
I put it there as a useful harmless example. Because this one is formatting only my files :) :)
When it is called this way format flag is doing its job. I found this on rescript forums. Many people have been banging their heads to the walls.